elastic/kibana
# [SecuritySolution] Add dashboard controls

## Summary

issue: Dashboard control was not rendered in Security dashboard page.


![portable-dashboard-controls](https://github.com/elastic/kibana/assets/6295984/4337046e-57d2-4f25-9c30-a987e59f79f3)



**Steps to verify:**
1. Import this dashboard to /app/management/kibana/object
[dashboard.zip](https://github.com/elastic/kibana/files/12163666/dashboard.zip)
2. Go to Security dashboard list and select this dashboard.
3. Observe the control should be there.

<img width="2546" alt="Screenshot 2023-07-25 at 17 43 11" src="https://github.com/elastic/kibana/assets/6295984/ba4e9c09-19b7-40ed-ab22-4835c6b3c765">


### Checklist

Delete any items that are not applicable to this PR.
